config/config_test.go

```diff
@@ -116,9 +116,11 @@ func TestNoServiceNameFromEnv(t *testing.T) {
 
 func TestSamplerConfigFromEnv(t *testing.T) {
 	// prepare
+	testMgrHostPort := "http://themaster:80"
```

Comment:
The previous version of the test didn't have the port. Usually, it's better to leave this as it was, to make sure the previous behavior is kept.